python/mdvtools/llm/code_manipulation.py

```
@@ -263,19 +128,27 @@ def patch_viewname(code: str, project: MDVProject):
-        view_name = name_match.group(1)
-        print(f"test: {view_name}")
-        # try to escape e.g. any quotes or other special characters in the view_name...
-        # this seems to be a somewhat accepted method (at least, for this particular local problem)
-        # view_name = json.dumps(view_name) # this will be done in the patch_viewname function
-        return view_name
+    try:
+        tree = ast.parse(code)
+        for node in ast.walk(tree):
+            if isinstance(node, ast.Assign):
+                # We are looking for a simple assignment, e.g. view_name = "..."
+                if len(node.targets) == 1 and isinstance(node.targets[0], ast.Name) and node.targets[0].id == 'view_name':
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)